easybuild/easyconfigs/l/libcotp/libcotp-1.2.3-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,36 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'libcotp'
+version = '1.2.3'
+
+homepage = 'https://github.com/paolostivanin/libcotp'
+description = """C library that generates TOTP and HOTP according to RFC-6238"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/paolostivanin/libcotp/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['f122aea37337cdfcdaa1ce979ecfb5177ad7dcb273caf2d6056820f1dbf92e93']  # v1.2.3.tar.gz
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+    ('libgcrypt', '1.9.2'),
+    ('libbaseencode', '1.0.11'),
+    ('pkg-config', '0.29.2'),
+    ('binutils', '2.35'),
+]
+
+parallel = 1
+
+separate_build_dir = True
+
+sanity_check_paths = {
+    'files': ['lib/libcotp.%s' % SHLIB_EXT],
+    'dirs': ['lib'],
```

Comment:
see previous comment

Reply:
That makes sense. I have changed it in both occasions to `include`.

Reply:
@Micket In light there has some time gone, do you want me to updated that to a newer toolchain?